doc/tutorials/setup_logical_partition_scenario_on_kbl_nuc_i7.rst

```diff
@@ -0,0 +1,154 @@
+.. _setup_logical_partition_scenario_on_kbl_nuc_i7:
+
+Guidelines of Setup Logical Partition Scenario on KBL NUC i7
+############################################################
+
+ACRN hypervisor supports logic partition scenario, in which the User OS (such as Clear Linux) running in a pre-launched VM can bypass the ACRN hypervisor and directly access isolated PCI devices. The following guidelines provide step by step instructions on how to set up the ACRN hypervisor logical partition scenario on KBL NUC i7 boards running two pre-launched VMs. The logical partition scenario on KBL NUC i7 is shown in :numref:`logical_partition_scenario_on_kbl_nuc_i7`.
```

Comment:
There is a mix of `logic partition` and `logical partition` in the text, can you use `logical` throughout the doc?

Reply:
fix at doc v2 version